.env.template

```diff
@@ -1,6 +1,13 @@
-THOTH_CEPH_BUCKET=
-THOTH_CEPH_BUCKET_PREFIX=
+# source ceph
+THOTH_DEPLOYMENT_NAME="ocp-test"
+
+THOTH_S3_ENDPOINT_URL="https://s3.upshift.redhat.com/
+THOTH_CEPH_BUCKET="thoth"
+THOTH_CEPH_BUCKET_PREFIX="data"
 THOTH_CEPH_KEY_ID=
 THOTH_CEPH_SECRET_KEY=
-THOTH_DEPLOYMENT_NAME=
-THOTH_S3_ENDPOINT_URL=
+
+# dest ceph
+THOTH_DOCUMENT_SYNC_DST="s3://thoth/data/test"
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
```

Comment:
Assuming smaug as target, we would have : 
`AWS_ENDPOINT_URL=https://s3-openshift-storage.apps.smaug.na.operate-first.cloud` correct ?
(This env var does not work, override the url by env var is not yet implemented in either boto3 or aws-cli, but the value are correct ?)
basing this on https://console-openshift-console.apps.smaug.na.operate-first.cloud/k8s/ns/thoth-infra-prod/configmaps/ceph

Reply:
The url is correct, i would update the documentation on where to place the endpoint as well.

Reply:
Or maybe we could put everything in THOTH_DOCUMENT_SYNC_DST (like `https://s3-openshift-storage.apps.smaug.na.operate-first.cloud/thoth/what/ever`).

The lack of a standard env var in aws/boto3 is quite a pain :/

Reply:
yeah this is a pain, if we are using the `aws cli` then we can rely on `--endpoint `.
However as I saw with your pr, it using boto3 and python module,maybe we can use the env var for this.

for now I have include a note for current setup.

Reply:
I can add an `--endpoint` flag with a env fallback to the command, with click it's quite trivial. (In fact I have a local commit to do just that that I used for testing).

Reply:
that would be great, seems like a convenient option as of now.

Reply:
Ok I'll add that in #37